shared/src/types/entities/apiClient/api.ts

```diff
@@ -212,6 +212,14 @@ export interface KeyValuePair {
   isEnabled: boolean;
   type?: string; // added for special identifiers like auth
   description?: string;
+  dataType?: ValueType;
+}
+
+export enum ValueType {
```

Comment:
let's rename to `KeyValueDataType` - agree it seems wordy, but feels consistent with how the other types are defined here